tests/conftest.py

```diff
@@ -276,12 +277,22 @@ def DummyMixedCoilSet(tmpdir_factory):
 
     tf_coil = FourierPlanarCoil(current=3, center=[2, 0, 0], normal=[0, 1, 0], r_n=[1])
     tf_coilset = CoilSet.linspaced_angular(tf_coil, n=4)
+    tf_coilset = CoilSet(tf_coilset.coils, NFP=tf_coilset.NFP, sym=True)
```

Comment:
Why do this? if you set `sym=True` but pass in these 4 coils, it essentially doubles the magnetic field by placing the symmetry coils on top of the other coils, and the min dist btwn coils in this coilset is now 0. I think we don't want this

Reply:
Hmm I remember @ddudt  requesting to pass in `sym=True` for this coilset to test symmetry but maybe I misunderstood what he said

Reply:
I think you can only pass in the first two of the coils, then set `sym=True` or something like that,so it uses symmetry to get the other two coils

Reply:
hm or maybe not, I will think but you should be able to make the same coilset using `sym=True` and `NFP` I think

Reply:
```python
from desc.coils import CoilSet, FourierPlanarCoil
tf_coil = FourierPlanarCoil(current=3, center=[2, 0, 0], normal=[0, 1, 0], r_n=[1])
tf_coilset = CoilSet.linspaced_angular(tf_coil, n=4)
tf_coilset1 = CoilSet(tf_coilset.coils[0], NFP=2, sym=True)```

@ddudt why does this not work? when I do compute position I find two of the coils are on top of eachother, when I would expect there to b3 4 distinct coils

Reply:
```python
tf_coil = FourierPlanarCoil(current=3, center=[2, 0, 0], normal=[0, 1, 0], r_n=[1])
tf_coil.rotate(angle=np.pi/4)
tf_coilset1 = CoilSet(tf_coil, NFP=2, sym=True)
```
this works @kianorr